src/convert.jl

```diff
@@ -222,22 +222,48 @@ function sexp2julia(rv::RVEC)
     end
 end
 
-function sexp2julia(rl::RList)
+function sexp2julia(rl::RList; metadata::Bool=true)
     if isdataframe(rl)
         # FIXME add force_missing option to control whether always convert to Union{T, Missing}
-        DataFrame(Any[isa(col, RAltRep) ? sexp2julia(col) : jlvec(col, false) for col in rl.data],
-                  identifier.(names(rl)), makeunique=true)
+        cols = Any[isa(col, RAltRep) ? sexp2julia(col) : jlvec(col, false) for col in rl.data]
+        nms = identifier.(names(rl))
+        obj = DataFrame(cols, nms, makeunique=true)
+        if metadata
+            for (key, val) in pairs(rl.attr)
+                # skip already processed system attributes
+                if key in ("names", "class")
+                    continue
+                elseif key in ("comment", "label")
+                    metadata!(obj, key, sexp2julia(val), style=:note)
```

Comment:
```suggestion
                    metadata!(obj, key, sexp2julia(val; metadata=metadata), style=:note)
```
I'd be explicit about metadata propagation here.